web/src/stream_settings_components.js

```diff
@@ -230,39 +180,26 @@ export function unsubscribe_from_private_stream(sub) {
         display_stream_archive_warning: sub_count === 1 && invite_only,
     });
 
-    function unsubscribe_from_stream() {
-        let $stream_row;
-        if (overlays.streams_open()) {
-            $stream_row = $(
-                "#channels_overlay_container div.stream-row[data-stream-id='" +
-                    sub.stream_id +
-                    "']",
-            );
-        }
-
-        ajaxUnsubscribe(sub, $stream_row);
-    }
-
     confirm_dialog.launch({
         html_heading: $t_html(
             {defaultMessage: "Unsubscribe from <z-link></z-link>?"},
             {"z-link": () => stream_name_with_privacy_symbol_html},
         ),
         html_body,
-        on_click: unsubscribe_from_stream,
+        on_click: () => ajaxUnsubscribe(sub),
     });
 }
 
-export function sub_or_unsub(sub, $stream_row) {
```

Comment:
`sub_or_unsub` is still being called with `$stream_row` in other files. Please have a look at those files and remove whatever code is necessary.

Reply:
Done!